docs/architecture/metadata/workflow.md

```diff
@@ -0,0 +1,30 @@
+## Live Reloading of a Data Page
+
+When modifying a YAML file for a Data page, executing the `etl ... --grapher` command to visualize updates can be inefficient. The following steps outline a streamlined approach for rapid development and testing.
+
+### Example Step
+For illustration, the step `grapher/gcp/2023-07-10/global_carbon_budget` is considered. The corresponding YAML file, `global_carbon_budget.meta.yml`, is assumed to be edited within the grapher channel and previewed on a [staging Data page](http://staging-site-mojmir/admin/datapage-preview/738081).
+
+### Initial Configuration
+- Add `DEBUG=1` to your `.env.myname` file and any other `.env.*` files in use to expedite ETL operations.
+
+### File Monitoring
+- Execute the ETL command with the `--watch` flag. This monitors the YAML file for changes and automatically re-executes the corresponding step.
+
+### Data Filtering
+- If the dataset contains numerous indicators, consider using `GRAPHER_FILTER=consumption_emissions_per_capita` to filter the data to only relevant variables. This is optional for smaller datasets.
+
+### Optional Optimization
+- Include the `--only` flag in the command to further improve performance by avoiding dependency checks.
+
+### Command Summary
+```bash
+ENV=.env.myname GRAPHER_FILTER=consumption_emissions_per_capita etl grapher/gcp/2023-07-10/global_carbon_budget --grapher --watch --only
+```
+NOTE: Working on your local grapher instead of a remote grapher will also reduce upload times. One option is to work on local grapher to begin with, and move to your staging server at a later point to be able to share the result with other colleagues.
+
```

Comment:
Use admonition instead

```suggestion
!!! note 
    Working on your local grapher instead of a remote grapher will also reduce upload times. One option is to work on local grapher to begin with, and move to your staging server at a later point to be able to share the result with other colleagues.

```